SocialCareCaseViewerApi/V1/Controllers/FormSubmissionController.cs

```diff
@@ -116,7 +116,7 @@ public IActionResult FinishSubmission(string submissionId, [FromBody] FinishCase
 
             try
             {
-                _formSubmissionsUseCase.ExecuteFinishSubmission(submissionId, request);
+                _formSubmissionsUseCase.ExecuteUpdateSubmission(submissionId, request);
                 return NoContent();
```

Comment:
i feel like when i've looked at this api endpoint before it returns a status code but no body. it would be good if it returned the newly updated submission object so the frontend can immediately show those changes, rather than have to query it a second time.

Reply:
We can add that, should be pretty simple